src/Eff/TH.hs

```diff
@@ -81,6 +81,7 @@ getDeclName = mkName . overFirst toLower . nameBase
 -- | Builds a function definition of the form @x a b c = send $ X a b c@.
 genDecl :: Con -> Q Dec
 genDecl (ForallC _ _ con) = genDecl con
+genDecl (NormalC cName tArgs') = genDecl $ GadtC [cName] tArgs' undefined
```

Comment:
It's okay that this is `undefined` since that parameter is never inspected (line 85).